modules/account-lib/package.json

```diff
@@ -39,6 +39,7 @@
     "@hashgraph/sdk": "1.4.6",
     "@stablelib/hex": "^1.0.0",
     "@stablelib/sha384": "^1.0.0",
+    "@stacks/transactions": "1.2.4",
```

Comment:
Can we use stacks/transactions 1.3.0 - re: this message from the blockstacks team: 
```
Hey @Wesley Graham! wanted to let you know that we just released version 1.3.0 of @stacks/transactions which includes support for uncompressed keys. Please let me know if you have any further issues with this!
```

Reply:
I've updated the dependency to 1.3.0.